src/bound.jl

```diff
@@ -7,6 +7,8 @@ and [`EqualTo`](@ref).
 """
 abstract type Bound end
 
+Base.length(x::Bound) = 1 # primarily for convenience in variables(...)
```

Comment:
no one is going to see the comment here, move the comment to Bound abstract type above.

The reason is "to make broadcasting work by default" instd of convenience.